apps/web/app/(app)/onboarding/OnboardingBulkUnsubscriber.tsx

```
@@ -0,0 +1,140 @@
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableHeader,
+  TableRow,
+} from "@/components/ui/table";
+import {
+  NewsletterStatsQuery,
+  NewsletterStatsResponse,
+} from "@/app/api/user/stats/newsletters/route";
```

Biome

[error] All these imports are only used as types.

Importing the types with import type ensures that they are removed by the transpilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)